src/Calculator/Controls/OverflowTextBlock.cpp

```diff
@@ -32,6 +32,7 @@ void OverflowTextBlock::OnApplyTemplate()
 {
     UnregisterEventHandlers();
 
+    // NEED TO ADD ANOTHER MEMBER VARIABLE TO KEEP TRACK OF MAIN RESULT SO THAT WE CAN USE ITS WIDTH IN SCROLLING CALCULATIONS
```

Comment:
>// NEED TO ADD ANOTHER MEMBER VARIABLE TO KEEP TRACK OF MAIN RESULT SO THAT WE CAN USE ITS WIDTH IN SCROLLING CALCULATIONS [](start = 4, length = 122)

Do you still need to do this? If not, remove the comment.

Reply:
Resolved.